cusignal_conda_env.yml

```diff
@@ -1,25 +1,17 @@
 name: cusignal
 channels:
   - nvidia
-  - rapidsai-nightly
   - conda-forge
   - defaults
-  - pytorch
+  - numba
 dependencies:
-  - pytorch
-  - cudf
-  - cuml
-  - cugraph
   - scipy
   - numpy
-  - pandas
-  - cudatoolkit=10.0
+  - cudatoolkit>=9.2,<10.2
   - boost
-  - jupyterlab
   - matplotlib
-  - nodejs
   - numba
   - pip
   - pytest
   - pip:
-    - git+https://github.com/cupy/cupy.git
+    - cupy
```

Comment:
You should be able to use conda cupy now. Check the cuDF environment yaml I believe we did it there

Reply:
https://anaconda.org/conda-forge/cupy

Reply:
Great -- Thanks for the tip, @beckernick! I'll need to move the conda-forge channel up in priority, as a conda install cupy  gave me v6.0.0. I'll add version limits too.